src/Components/PresentationalComponents/Header/Header.js

```diff
@@ -18,7 +18,13 @@ const Header = ({ title, actions, breadcrumbs, showBreadcrumb, children, actions
                     <PageHeaderTitle title={title} />
                 </SplitItem>
 
-                <SplitItem isFilled></SplitItem>
+                {[].concat(labels).map((item, index) => (
```

Comment:
@leSamo I'm sorry I didn't get it, what's the purpose of it?

Reply:
In the future we will need to be able to pass multiple chips to the header

![Screenshot from 2021-02-03 14-29-05](https://user-images.githubusercontent.com/8426204/106753629-35138d00-662c-11eb-8594-c18d2e470b0e.png)

Reply:
I see, thanks for the clarification